content/_snippets/data_types/currency_code.md

```diff
@@ -1,4 +1,6 @@
-There are two kinds of currency code in the XRP Ledger:
+The [`rippled` APIs](rippled-api.html) support two formats of currency code for issued currencies:
 
-* Three-character currency code. We recommend using all-uppercase [ISO 4217 Currency Codes](http://www.xe.com/iso4217.php). However, any combination of the following characters is permitted: all uppercase and lowercase letters, digits, as well as the symbols `?`, `!`, `@`, `#`, `$`, `%`, `^`, `&`, `*`, `<`, `>`, `(`, `)`, `{`, `}`, `[`, `]`, and <code>&#124;</code>. The currency code `XRP` (all-uppercase) is reserved for XRP and cannot be used by issued currencies.
-* 160-bit hexadecimal values, such as `0158415500000000C1F76FF6ECB0BAC600000000`, according to the XRP Ledger's internal [Currency Format](currency-formats.html). This representation is uncommon.
+- **[Standard Currency Codes](currency-formats.html#standard-currency-codes):** As a 3-character string such as `"EUR"` or `"USD"`.
+- **[Nonstandard Currency Codes](currency-formats.html#nonstandard-currency-codes):** As a 160-bit hexadecimal string, such as `"0158415500000000C1F76FF6ECB0BAC600000000"`. This is uncommon.
```

Comment:
I don't know how uncommon it is today. It depends entirely on how people end up using the XRP Ledger.

As for why you'd use it, it would be if you wanted to encode more information into the currency code than just three characters.

For example, if you wanted to trade securities with longer ticker symbols on the XRP Ledger, you could define a standard that translates those ticker symbols (client-side) into the raw hex format for use with the ledger. (Note, there may be legal challenges with doing that, but that's besides the point.) Back in 2014 or so, ripple-lib supported currency codes that had interest built in so that Gold Bullion International could issue gold on the XRP Ledger with [explicit demurrage fees](https://xrpl.org/demurrage.html). There are other possibilities, too, but what it comes down to is, you want currency codes that carry more than 3 letters' worth of information.